### ISSUE TITLE
Webview2 crashes when calling document.write on newly opened window

### ISSUE THREAD

**Issue by attilamn** [MicrosoftEdge/WebView2Feedback#2361]
**Description**
Webview2 crashes when calling document.open, document.write (with HTML tag), document.close on newly opened window when handling NewWindowRequested event and assigning new window to created Webview2.

In CoreWebView2_ProcessFailed event, it fires twice with the following:
ExitCode = -1073741819 
ProcessFailedKind = BrowserProcessExited 
Reason = Unexpected

**Version**
Runtime: 100.0.1185.36
Framework: WPF
OS: Win10

**Repro Steps**
NewWindowRequested event:
```
var deferral = e.GetDeferral();

e.Handled = true;
MainWindow mw = new MainWindow
{
    Deferral = deferral,
    Args = e
};

mw.Show();
```
CoreWebView2InitializationCompleted event:
```
if (Deferral != null)
{
    Args.NewWindow = wv.CoreWebView2;
    Deferral.Complete();
}
```

HTML:
```
<html>
<head>
<title>Test</title>
</head>
<body>
<button onclick="opentwo()">Open</button>

<script type="text/javascript">
    function opentwo() {
        var two = window.open('');
        two.document.open();
        two.document.write('<html><head><title>test</title></head><body>this is the body</body></html>')
        two.document.close();
    }
</script>
</body>
</html>
```

**Comment by attilamn** [comment]
I have created a simple project to repro the behavior - https://github.com/attilamn/webview2-document-write-test

**Comment by david-risney** [comment]
Cagri this looks like a bug you've already fixed (38613250). I was looking into this on the [corresponding StackOverflow question ](https://stackoverflow.com/a/71887645/3943588) without realizing there was also this GitHub issue =)
